Examples/Kinova/SystemIdentification/ExcitingTrajectories/src/EndEffectorConditionalNumberOptimizer.cpp

```diff
@@ -0,0 +1,239 @@
+#include "EndeffectorConditionNumberOptimizer.h"
```

Comment:
Please remove this since this is supposed to be in sys_id_dev. You should not develop different features in one branch.